1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -418,22 +419,30 @@ A YAML nested list (`-`) of DNS resolvers for your DC/OS cluster nodes. You can
 **Caution:** If you set the `resolvers` parameter incorrectly, you will permanently damage your configuration and have to reinstall DC/OS.
 
 ### rexray_config
-The <a href="https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/" target="_blank">REX-Ray</a> configuration method for enabling external persistent volumes in Marathon. REX-Ray is a storage orchestration engine. The following is an example configuration.
-
-    rexray_config:
-      rexray:
-        loglevel: info
-        modules:
-          default-admin:
-            host: tcp://127.0.0.1:61003
-        storageDrivers:
-        - ec2
+The <a href="https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/" target="_blank">REX-Ray</a> configuration for enabling external persistent volumes in Marathon. REX-Ray is a storage orchestration engine. The following is an example configuration.
+
```

Comment:
I wonder if we should keep the `rexray_config:` line to indicate how the config is supposed to look in the user's config.yaml. 

```
rexray_config:
  rexray:
    loglevel: info
    service: ebs
  libstorage:
    integration:
      volume:
    ...
```